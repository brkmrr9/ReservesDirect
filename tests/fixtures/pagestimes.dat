#148, #153, #191, #207, #150
( 1 page )
( 8 pages )
( p100 )
( pp. 10 - 13 )
( pp. 74 - 85 )
( pp. H1 - H4 )
(1 of 102)
(10 pages)
(10 pg)
(1:48)
(8:44)
(Selections)
-
- / -
-- - --
/
/ 1
/ 10
0 / 0
00 / 00
00. 195-204 (9 of 559)
01/16-03/30
0243 - -279
0:32
0:49 and 1:17
1
1  / 14
1 -
1 - 1
1 - 10 (10 of 2705)
1 - 7
1 - 6 (plus figures)
1 page
1 videocassette (VHS) (8 min.) : sd., col. ; 1/2 in
1'10
1, 5-13
1--22
1-10, 18-21, 25-27, 41-3, and 66-7
1-10, 97-122
1-10-21-39
1-11, +
1-11, 41-44, 78-87
1-12 (12 pp. of 1300 total)
1-13; 34-55
1-14, 111-129, 212-221, n 249-52, 274-78, 283-84
1-14, 27-49, 71-86, 109-143
1-15, 149-56
1-16, 74-100, 126-157
4-6 and 180-182
4-6, 273-278 (notes)
4-6, 334-367
4-6, 91-106
4-6, Notes
11-13, 198-202
11-13.0
1
1.1
104
11491159
- / -
-- - --
??
???
( 1 page )
( 15 pages )
( p100 )
( pp. 10 - 13 )
( pp. H1 - H4 )
( pp.106-110 )
(1 of 102)
(1:48)
(10 pages)
(10 pg)
(2)
(58 of 218)
(8 of 2 volumes)
(8 of 468)
(8 pages)
(8:45)
(pp.66-75)
(pp.71 - 86)
Selections
/
/ 1
/ 10
#148, #153, #191, #207, #150
0 / 0
0:38
0:49 and 1:17
00 / 00
00. 195-204 (9 of 559)
01/16-03/30
0243 - -279
1  / 14
1 -
1 - 1
1 - 10 (10 of 2705)
1 - 6 (plus figures)
1 page
1 videocassette (VHS) (8 min.) : sd., col. ; 1/2 in
1--22
1-10-21-39
1-10, 18-21, 25-27, 41-3, and 66-7
1-10, 97-122
1-11, +
1-11, 41-44, 78-87
1-12 (12 pp. of 1300 total)
1-13; 34-55
1-14, 111-129, 212-221, n 249-52, 274-78, 283-84
1-14, 27-49, 71-86, 109-143
1-15, 149-56
1-16, 74-100, 126-127
1-2, 1-2, 1-2
1-24 /
1-26-175-184, +
1-27 +notes
1-27, 154-162 / 162
1-27, 86-93, 116-123, 214-219
1-28 and 144-46
1-318 (excerpts)
1-39, title page
1-4, 38-54, 130-136
1-40
1-43 (43 of 1883)
1-43 + notes
1-6 / 576
1-7, 42-49, 123-128
1-78, 125-131, 134-144 (96 of 38966)
1-8 (8 of 289)
1-8-27-46
1-8, 19-37, 42-45, 67-71, 103-104
1-80, 287-98
1-9-14-26
1, 5-13
1.)  p. 171-181   2.)  p. 351-358
1'10
1/ 14
10  / 118
10 -
10 / c.150
10 of 181
10 pages total
10 pgs
10 pp.
10-10
10:07
10/11/2007
100 min.
100-117 / 224
100-117, 217-219, 231-246 (27 of 1280)
1009 - 1021
101 -
102,104,107 (3 of 968)
1021-1053 /
103-116, +
103-127 (24 of 640)
10363 - 10367
1039-1042.
107, 112-113, 131-132, 135, 243-244, 246-247, 349, 454-455, 661, 675, 888, 891, 930, 953, 960, 1026, 1033, 1041, 1050, 1103-1104, 1179-1181, 1205, 1209
108 min.
131-134, +
126–127
1099–104
11 / 100?
11-13, 198-202
11-13; 189-198
11-13; 30-37
11-13.
113-123 and 184-191
114-120, 124-127 (11 of 167)
114-121 (8 of 332 = 2%)
115 ff.-
119-151, 159-160 (notes)
12-27 (15 pp. of 1300 total)
1203 - 1207 & 1241
121-124 (Appendix A)
121-134-163-164
1218S-1219S
125-161; Notes:  279-289
126-127, 146-155
127-128, 184-198, and 210
128 p. illus. 21 cm
128-33; 411-15-
153-178 & notes
155-160, 161-165, 249-252 (15 of 500)
1552-1599, 1622-1645, 1688-1701 (86 of 2487)
156-157 (out of 275)
16-29, 64-69, 94-96, 145-149, 335-343 (38 of 1528)
17-19 and 34-46
17-19, 418-421
17-19; 76-105 (50 of 277)
4-6 and 180-182
4-6, 273-278 (notes)
4-6, 334-367
4-6, 91-106
4-6, Notes
9–17
Â´XVII-XXI
A-3
Act 2, Scene 7, Lines 138-165
Convention of Counties, October 12, 2009, Confimation done, June 21, 2714
Appendix 11.  pp 419-421
Article 13, pp. 83-87
Article 9, pp. 61- 65
asf
B03 -
Ch 10.  pp 283-285.
CH 4: pp. 91 - 111
Ch 5-6.  pp 212-214, 221-32
CH 8: pp.181-33
Ch. 1 and 2
Ch. 1 Sun, 9-14
Ch. 1-2, p. 2-40
Ch. 1, 3-5
ch. 10, p. 131-146
Ch. 10, pp. 71-76
Ch. 7: War, 190-195
Ch. 8 (pp. 199-202) and Notes (pp. 310-14)
Ch. 8, pp.190-189
Ch. 8: Water, 299-302
chap. 14
Chap. 2 pp. 25-27
Chapter 1
Chapter 1: pp. 1 - 4 (includes notes)
Chapter 10: pp. 160-164
CHAPTER 10: pp. 191- 197
Chapter 11.  pp399-402.
CHAPTER 12 pp. 137-139
Chapter 13.  pp487-490.
Err:509
i - 22
i-ix
i-xvi-1-6
III6-III10
Introduction; pp. 27 - 47 and 67 - 85
ix-104
ix-xiii
ix-xiv
ix-xxii (14 of 247)
ix-xxiii
ix-xxvii; 385-549
ix, 291 p. : ill. (some col.), col. maps ; 25 cm.
ix.-xxv.
January 24th
p. 8-9 plus
p. 8-9 plus Contents, Forward, & Intro.
p.A-1 to A-17 (17 of 411)
p.ix-xiv
p.ix-xvii and p.17-19
p.vii-xiv and p.198-201
p.xv-xxix
p104-107, 4p
pg:1042
pg:12
pg:1349
pg:i13 -i23
pg. A.31
pg. C.1
pgs. 1-10
pgs. 421-442.
plates 4-16
plp. 429-469
pp .xii-xvi, 291-307
pp 1 - 3
pp 1-10
pp 1-11.
pp.
pp.  (2 of 475)
pp.  1 - 14
pp. ?
pp. ??-?? (24 of 334)
pp. . 1560-1642
pp. ( of 333)
pp. (of 333)
pp. 1 - 13.
pp. 1 - 25 and pp. 225 - 235
pp. 1 (1)
pp. 1- 14 (14 of 1993)
pp. 1- 62 (62 of 6662)
pp. 1- 8
pp. 1-14 (14)
pp. 1-14, 141-169, 195-202, 203-204, 234-241, 247 (62 of 1273)
pp. 1-14, 195-206 (26 of 282)
pp. 23-38, 86-106 (37 of 2118)
pp. 1-14; 35-53 (32 of 182)
pp. 1-14.
pp. 1-148 (148 of 368)
pp. 1-16,109-111 (19 of ?)
pp. 1-18  (11)
pp. 1-3 (22 of 1 volume)
pp. 1-3 (23 of ??)
pp. 1-3, 22-36 (36 of 231)
pp. 1-3, 283-284(notes) (23 of 306)
pp. 1-3, 32-33, 58-65 (31 of 177)
pp. 1-3, 90-117,  119-133
pp. 1-3, notes, 229-231 (24 of 989)
pp. 1, 13-16 (5 of 1448)
pp. 1, 16, 44, 79, 114, 150, 186, 216, 256, 290, 312, 336, 362, 388
pp. 1, 3-17 (16 of 4493)
pp. 1,16,44,79,114,150,186,216,256,290,312,336,362,388 (14 of 4100)
pp. 1,2,21-62 (44 of 1756)
pp. 4-6 (18 of 311)
pp. 4-6 and pp. 469-521 (68 of 6481)
pp. 4-6; 19-27 (26)
pp. 4-6; 224-227
pp. 4-6; 32-47 (33 of 1257)
pp. A-4 - A-69
pp. A1- (3 pp)
pp. A4
pp. B-4 - B-45
pp. B1, B4
pp. B12-B13
pp. B147-156
pp. C-290-297
pp. C-4 - C-5
pp. C1
pp. C1 and C11
pp. D.1
pp. G4
pp. i - viii
pp. i-27
pp. i-iv; 17-19 (24 of 8224)
pp. II-102  -  II-124
pp. ii-viii, 29-34 (12 of 1150)
pp. iii, 1-39 (39 of 6139)
pp. iii, iv, 8 (3 of 180)
pp. iv-v; xv-xx; 1-8; 17-25 (24 of 807)
pp. p2018-2019
pp. pp. 1- 16
pp. S1-S14
pp. S2-S8, S11-S15, S20-S21
S103 - S126
Slides 21 - 22
1 - 10 (10 of 275)
00. 195-204 (9 of 559)
1 - 23 (23 of 432)
1-10 (10 of 203 = 5%)
1-11 (11 of 14)
1-12 (12 pp. of 1890 total)
1-21 (21 of 243 = 9%)
139-170 (31 out of 308 total)
156-157 (out of 275)
33-86, 96-98 (50 out of 890)
selected pages
selections
various
various chapters
ix
v
vii
various sections
3:02 of 69:29
2:39 of 69:10
10:41
5:57
2:59 of 35:55
2:50 of 71:59
3:48
3:23 of 71:59
5:22 of 72:02
13:44 of 40:36
6:43 out of 70:20

