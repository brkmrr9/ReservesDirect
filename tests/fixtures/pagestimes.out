, 5, 148, 153, 191, 207, 150
, 1, 
, 8, 
, 1, 100
, 4, 10-13
, 12, 74-85
, , ( pp. H1 - H4 )
102, 1, 
, 10, 
, 10, 
, 1:48, 
, 8:44, 
, , 
, , 
, , 
, , 
, , 
, , / 1
, , / 10
, , 
, , 
, , 00. 195-204 (9 of 559)
, , 01/16-03/30
, , 0243 - -279
, 0:32, 
, , 0:49 and 1:17
, 1, 1
14, 1, 
, , 1 -
, 1, 1-1
2705, 10, 1-10
, 7, 1-7
, , 1 - 6 (plus figures)
, 1, 
, , 1 videocassette (VHS) (8 min.) : sd., col. ; 1/2 in
, , 1'10
, 10, 1, 5-13
, , 1--22
, , 1-10, 18-21, 25-27, 41-3, and 66-7
, 36, 1-10, 97-122
, , 1-10-21-39
, , 1-11, +
, 25, 1-11, 41-44, 78-87
, , 1-12 (12 pp. of 1300 total)
, 35, 1-13, 34-55
, , 1-14, 111-129, 212-221, n 249-52, 274-78, 283-84
, 88, 1-14, 27-49, 71-86, 109-143
, 22, 1-15, 149-56
, 75, 1-16, 74-100, 126-157
, , 4-6 and 180-182
, 9, 4-6, 273-278
, 37, 4-6, 334-367
, 19, 4-6, 91-106
, , 4-6, Notes
, 8, 11-13, 198-202
, , 11-13.0
, 1, 1
, , 1.1
, 1, 104
, 1, 11491159
, , 
, , 
, , 
, , 
, 1, 
, 15, 
, 1, 100
, 4, 10-13
, , ( pp. H1 - H4 )
, 5, 106-110
102, 1, 
, 1:48, 
, 10, 
, 10, 
, 1, 2
218, 58, 
, , (8 of 2 volumes)
468, 8, 
, 8, 
, 8:45, 
, 10, 66-75
, 16, 71-86
, , 
, , 
, , / 1
, , / 10
, 5, 148, 153, 191, 207, 150
, , 
, 0:38, 
, , 0:49 and 1:17
, , 
, , 00. 195-204 (9 of 559)
, , 01/16-03/30
, , 0243 - -279
14, 1, 
, , 1 -
, 1, 1-1
2705, 10, 1-10
, , 1 - 6 (plus figures)
, 1, 
, , 1 videocassette (VHS) (8 min.) : sd., col. ; 1/2 in
, , 1--22
, , 1-10-21-39
, , 1-10, 18-21, 25-27, 41-3, and 66-7
, 36, 1-10, 97-122
, , 1-11, +
, 25, 1-11, 41-44, 78-87
, , 1-12 (12 pp. of 1300 total)
, 35, 1-13, 34-55
, , 1-14, 111-129, 212-221, n 249-52, 274-78, 283-84
, 88, 1-14, 27-49, 71-86, 109-143
, 22, 1-15, 149-56
, 45, 1-16, 74-100, 126-127
, 6, 1-2, 1-2, 1-2
, , 1-24 /
, , 1-26-175-184, +
, , 1-27 +notes
, 36, 1-27, 154-162
, 49, 1-27, 86-93, 116-123, 214-219
, , 1-28 and 144-46
, , 1-318 (excerpts)
, , 1-39, title page
, 28, 1-4, 38-54, 130-136
, 40, 1-40
1883, 43, 1-43
, , 1-43 + notes
576, 6, 1-6
, 21, 1-7, 42-49, 123-128
38966, 96, 1-78, 125-131, 134-144
289, 8, 1-8
, , 1-8-27-46
, 38, 1-8, 19-37, 42-45, 67-71, 103-104
, 91, 1-80, 287-98
, , 1-9-14-26
, 10, 1, 5-13
, , 1.)  p. 171-181   2.)  p. 351-358
, , 1'10
14, 1, 
118, 10, 
, , 10 -
, , 10 / c.150
181, 10, 
, 10, 
, 10, 
, , 10 pp.
, 1, 10-10
, 10:07, 
, , 10/11/2007
, , 100 min.
224, 18, 100-117
1280, 37, 100-117, 217-219, 231-246
, 13, 1009-1021
, , 101 -
968, 3, 102, 104, 107
, , 1021-1053 /
, , 103-116, +
640, 24, 103-127
, 5, 10363-10367
, 4, 1039-1042
, 31, 107, 112-113, 131-132, 135, 243-244, 246-247, 349, 454-455, 661, 675, 888, 891, 930, 953, 960, 1026, 1033, 1041, 1050, 1103-1104, 1179-1181, 1205, 1209
, , 108 min.
, , 131-134, +
, , 126–127
, , 1099–104
, , 11 / 100?
, 8, 11-13, 198-202
, 13, 11-13, 189-198
, 11, 11-13, 30-37
, 3, 11-13
, , 113-123 and 184-191
167, 11, 114-120, 124-127
332, 8, 114-121
, , 115 ff.-
, 35, 119-151, 159-160
, , 12-27 (15 pp. of 1300 total)
, , 1203 - 1207 & 1241
, , 121-124 (Appendix A)
, , 121-134-163-164
, , 1218S-1219S
, , 125-161; Notes:  279-289
, 12, 126-127, 146-155
, , 127-128, 184-198, and 210
, , 128 p. illus. 21 cm
, 9, 128-33, 411-15
, , 153-178 & notes
500, 15, 155-160, 161-165, 249-252
2487, 86, 1552-1599, 1622-1645, 1688-1701
275, 2, 156-157
1528, 37, 16-29, 64-69, 94-96, 145-149, 335-343
, , 17-19 and 34-46
, 7, 17-19, 418-421
277, 33, 17-19, 76-105
, , 4-6 and 180-182
, 9, 4-6, 273-278
, 37, 4-6, 334-367
, 19, 4-6, 91-106
, , 4-6, Notes
, , 9–17
, , Â´XVII-XXI
, , A-3
, , Act 2, Scene 7, Lines 138-165
, , Convention of Counties, October 12, 2009, Confimation done, June 21, 2714
, , Appendix 11.  pp 419-421
, , Article 13, pp. 83-87
, , Article 9, pp. 61- 65
, , asf
, , B03 -
, , Ch 10.  pp 283-285.
, , CH 4: pp. 91 - 111
, , Ch 5-6.  pp 212-214, 221-32
, , CH 8: pp.181-33
, , Ch. 1 and 2
, , Ch. 1 Sun, 9-14
, , Ch. 1-2, p. 2-40
, , Ch. 1, 3-5
, , ch. 10, p. 131-146
, , Ch. 10, pp. 71-76
, , Ch. 7: War, 190-195
, , Ch. 8 (pp. 199-202) and Notes (pp. 310-14)
, , Ch. 8, pp.190-189
, , Ch. 8: Water, 299-302
, , chap. 14
, , Chap. 2 pp. 25-27
, , Chapter 1
, , Chapter 1: pp. 1 - 4 (includes notes)
, , Chapter 10: pp. 160-164
, , CHAPTER 10: pp. 191- 197
, , Chapter 11.  pp399-402.
, , CHAPTER 12 pp. 137-139
, , Chapter 13.  pp487-490.
, , Err:509
, , i - 22
, 9, i-ix
, , i-xvi-1-6
, , III6-III10
, , Introduction; pp. 27 - 47 and 67 - 85
, , ix-104
, 5, ix-xiii
, 6, ix-xiv
, , ix-xxii (14 of 247)
, 15, ix-xxiii
, , ix-xxvii; 385-549
, , ix, 291 p. : ill. (some col.), col. maps ; 25 cm.
, , ix.-xxv.
, , January 24th
, , p. 8-9 plus
, , p. 8-9 plus Contents, Forward, & Intro.
, , p.A-1 to A-17 (17 of 411)
, 6, ix-xiv
, , p.ix-xvii and p.17-19
, , p.vii-xiv and p.198-201
, 15, xv-xxix
, 5, 104-107, 4
, 1, 1042
, 1, 12
, 1, 1349
, , pg:i13 -i23
, , pg. A.31
, , pg. C.1
, , pgs. 1-10
, , pgs. 421-442.
, , plates 4-16
, , plp. 429-469
, , pp .xii-xvi, 291-307
, 3, 1-3
, 10, 1-10
, 11, 1-11
, , pp.
, , pp.  (2 of 475)
, 14, 1-14
, , pp. ?
, , pp. ??-?? (24 of 334)
, , pp. . 1560-1642
, , pp. ( of 333)
, , pp. (of 333)
, 13, 1-13
, , pp. 1 - 25 and pp. 225 - 235
, , pp. 1 (1)
1993, 14, 1-14
6662, 62, 1-62
, 8, 1-8
, 14, 1-14
1273, 62, 1-14, 141-169, 195-202, 203-204, 234-241, 247
282, 26, 1-14, 195-206
2118, 37, 23-38, 86-106
182, 33, 1-14, 35-53
, 14, 1-14
368, 148, 1-148
, 19, 1-16, 109-111
, 18, 1-18
, , pp. 1-3 (22 of 1 volume)
, , pp. 1-3 (23 of ??)
231, 18, 1-3, 22-36
, 5, 1-3, 283-284
177, 13, 1-3, 32-33, 58-65
, 46, 1-3, 90-117, 119-133
, , pp. 1-3, notes, 229-231 (24 of 989)
1448, 5, 1, 13-16
, 14, 1, 16, 44, 79, 114, 150, 186, 216, 256, 290, 312, 336, 362, 388
4493, 16, 1, 3-17
4100, 14, 1, 16, 44, 79, 114, 150, 186, 216, 256, 290, 312, 336, 362, 388
1756, 44, 1, 2, 21-62
311, 18, 4-6
, , pp. 4-6 and pp. 469-521 (68 of 6481)
, 12, 4-6, 19-27
, 7, 4-6, 224-227
1257, 19, 4-6, 32-47
, , pp. A-4 - A-69
, , pp. A1- (3 pp)
, , pp. A4
, , pp. B-4 - B-45
, , pp. B1, B4
, , pp. B12-B13
, , pp. B147-156
, , pp. C-290-297
, , pp. C-4 - C-5
, , pp. C1
, , pp. C1 and C11
, , pp. D.1
, , pp. G4
, 8, i-viii
, , pp. i-27
, , pp. i-iv; 17-19 (24 of 8224)
, , pp. II-102  -  II-124
, , pp. ii-viii, 29-34 (12 of 1150)
6139, 40, iii, 1-39
180, 3, iii, iv, 8
, , pp. iv-v; xv-xx; 1-8; 17-25 (24 of 807)
, , pp. p2018-2019
, , pp. pp. 1- 16
, , pp. S1-S14
, , pp. S2-S8, S11-S15, S20-S21
, , S103 - S126
, 2, 21-22
275, 10, 1-10
, , 00. 195-204 (9 of 559)
432, 23, 1-23
203, 10, 1-10
14, 11, 1-11
, , 1-12 (12 pp. of 1890 total)
243, 21, 1-21
, , 139-170 (31 out of 308 total)
275, 2, 156-157
890, 57, 33-86, 96-98
, , 
, , 
, , 
, , 
, 1, ix
, 1, v
, 1, vii
, , 
69:29, 3:02, 
69:10, 2:39, 
, 10:41, 
, 5:57, 
35:55, 2:59, 
71:59, 2:50, 
, 3:48, 
71:59, 3:23, 
72:02, 5:22, 
40:36, 13:44, 
70:20, 6:43, 
